src/components/AssignmentTexter/Controls.jsx

```diff
@@ -786,7 +786,6 @@ export class AssignmentTexterContactControls extends React.Component {
   }
 
   renderMessagingRowSendSkip(contact) {
-    console.log("this.props", this.props);
```

Comment:
Note to reviewers: this removes an unnecessary and verbose console.log.

Reply:
this makes merges harder, so please don't do this when the file is unmodified otherwise in the PR.